Sources/SourceKitLSP/Swift/DocumentSymbols.swift

```diff
@@ -228,6 +276,19 @@ fileprivate extension SyntaxProtocol {
   var rangeWithoutTrivia: Range<AbsolutePosition> {
     return positionAfterSkippingLeadingTrivia..<endPositionBeforeTrailingTrivia
   }
+
+  /// Whether this is a top-level constant or a member of a type, ie. if this is not a local variable.
+  var isMemberOrTopLevelDeclaration: Bool {
+    if self.parent?.is(MemberBlockItemSyntax.self) ?? false {
+      return true
+    } else if let codeBlockItem = self.parent?.as(CodeBlockItemSyntax.self),
+      codeBlockItem.parent?.parent?.is(SourceFileSyntax.self) ?? false
+    {
+      return true
+    } else {
+      return false
+    }
```

Comment:
Do you *really* find that easier to read than eg.
```
    if self.parent?.is(MemberBlockItemSyntax.self) ?? false {
      return true
    }

    if let codeBlockItem = self.parent?.as(CodeBlockItemSyntax.self),
      codeBlockItem.parent?.parent?.is(SourceFileSyntax.self) ?? false {
      return true
    }

    return false
```

Reply:
I don’t have super strong opinions. Changed it